frontend/src/book-read/components/review-update.page.jsx

```diff
@@ -0,0 +1,35 @@
+import ReviewMutationComponent from "./shared/review-mutation";
+import {
+  useUpdateReviewMutation,
+  useGetReviewByBookQuery,
+} from "../store/apis/review";
+import { useParams } from "react-router-dom";
+import { renderValidationErrorsObject } from "../../shared";
+
+export default function ReviewCreatePage() {
+  const { bookId } = useParams();
+  const { data: review } = useGetReviewByBookQuery(bookId);
+
+  const [updateReview, updateResults] = useUpdateReviewMutation();
+
+  const handleSubmit = (review) => {
+    updateReview(review);
+  };
+
+  let error;
+  if (updateResults.isError) {
+    error = renderValidationErrorsObject(updateResults.error.data.errors);
+  }
+
+  return (
+    <ReviewMutationComponent
+      mutationTitle="Update"
+      onSubmit={handleSubmit}
+      review={review}
+      // isLoading={updateResults.isLoading}
```

Comment:
If this code is unused, it should be removed.

Reply:
Reused